functions/src/events/events.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * Events
+    Workshop 14
+        Title
+        Description
+        Instructions: []
+        Host: uid
+        Active: true
+        Dates: {
+	        Start: timestamp
+	        End: timestamp
+        }
+        Tags: [],
+        Category: [],
+        Online: boolean
+        Stats: {
+            Likes: []
+            Rsvp: []
+            people_who_showed_up:
+        people_who_stayed_till_the_end:
+        }
+        Location: {
+            Geo: {
+                Geopoint: geopoint,
+                Geohash: geohash
+        },
+        Address: {
+            fullAddress: string,
+            fullStreet: string,
+            streetNum: string,
+            Street: string,
+            City: string,
+            State: string?,
+            Country: String
+        }
+    }
+
+ */
+
+import { firestore } from "../admin/admin";
+
+interface event {
+    title: string;
+    description: string;
+    instructions: [];
+    host: number;
+    active: boolean;
+    tags: [];
+    category: [];
+    online: boolean;
+    stats: {"likes" : [], "rsvp" : [], "showedUp" : number, "stayed" : number};
+    dates: {};
+    location: {"geo" : [], "address": { "fullAddress" : string
+                                        "fullStreet" : string
+                                        "streetNum" : number
+                                        "street" : string
+                                        "city" : string
+                                        "state" : string
+                                        "zip" : number
+                                        "country" : string
+                                       }
+    }
+    
+}
+
+exports.createEvent = async (request: any, response: any) => {
```

Comment:
```suggestion
const createEvent = async (request: any, response: any): Promise<void> => {
```

Use this format for the functions, you can export them all at once at the end of the file
Check the `feature/roles` branch for more examples